src/main/resources/native/ios/AppDelegate.m

```diff
@@ -109,6 +109,32 @@ - (void)applicationWillTerminate:(UIApplication *)application {
     gvmlog(@"[UIAPP] applicationWillTerminate");
 }
 
+- (BOOL)application:(UIApplication *)app
+            openURL:(NSURL *)url
+            options:(NSDictionary<NSString *, id> *)options {
+
+    gvmlog(@"[UIAPP] application_openURL_options");
+
+    [[NSNotificationCenter defaultCenter]
+        postNotificationName:@"UIApplicationOpenURLNotification"
+        object:self userInfo:[NSDictionary dictionaryWithObject:url forKey:@"URL"]];
+
+    return YES;
```

Comment:
Not sure what the best approach would be for knowing whether or not the URL notification was actually handled.. or if it matters.